docs/fundamentals/snippets/volumes/VolumeMounts.AppHost/Program.WithVolume.cs

```diff
@@ -0,0 +1,13 @@
+﻿internal partial class Program
+{
+    private static IResourceBuilder<SqlServerDatabaseResource> WithVolume(IDistributedApplicationBuilder builder)
+    {
+        // <mount>
```

Comment:
Is this the correct id?

Reply:
Fixed. I've reviewed all the IDs and ensured they match the :::code::: links.